<?xml version="1.0"?>

<launch>
  <arg name="uri" default="radio://0/80/250K" />
  <arg name="joy_dev" default="/dev/input/js0" />

  <include file="$(find crazyflie_driver)/launch/crazyflie_server.launch">
  </include>

  <group ns="crazyflie">
    <!--<param name="crazyflie/pid_rate/yaw_kp" value="200"/>-->
    <include file="$(find crazyflie_driver)/launch/crazyflie_add.launch">
      <arg name="uri" value="$(arg uri)" />
      <arg name="tf_prefix" value="crazyflie" />
      <arg name="enable_logging" value="True" />
    </include>

    <node name="joy" pkg="joy" type="joy_node" output="screen" >
      <param name="dev" value="$(arg joy_dev)" />
    </node>

    <include file="$(find crazyflie_demo)/launch/xbox360.launch">
    </include>

    <node name="crazyflie_demo_controller" pkg="crazyflie_demo" type="controller.py" output="screen">
    </node>
  </group>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find crazyflie_demo)/launch/crazyflie.rviz" />

  <!-- <node pkg="rqt_plot" type="rqt_plot" name="rqt_plot" args="/crazyflie/temperature/temperature"/> -->

  <node pkg="rqt_plot" type="rqt_plot" name="rqt_plot2" args="/crazyflie/battery"/>

  <node pkg="rqt_plot" type="rqt_plot" name="rqt_plot3" args="/crazyflie/rssi"/>
</launch>
